client.go

```diff
@@ -0,0 +1,63 @@
+package nomadclient
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"fmt"
+	dphttp "github.com/ONSdigital/dp-net/http"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+type Nomad struct {
+	Client dphttp.Clienter
+	URL string
+	// Name needed for checker later.
+	// Name string
+}
+
+func NewClient(NomadEndpoint, NomadCACert string, NomadTLSSkipVerify bool) (*Nomad, error){
+
+	dpHTTPClient := *dphttp.DefaultClient
+
+	if strings.HasPrefix(NomadEndpoint, "https://") {
```

Comment:
Consider making `"https://"` a constant - otherwise you are creating the same var over and over again. Admittedly in this case it's a trivial amount of memory each time but I think it's a good mind set to have to question if you are creating objects, variables unnecessarily.